pubtools/_pulp/tasks/push/command.py

```diff
@@ -112,7 +113,9 @@ def add_phase(klass, **kwargs):
                 update_push_items=collect_phase.in_queue.put,
                 publish_with_cache_flush=self.publish_with_cache_flush,
             )
-            phases.append(klass(**kwargs))
+            phase_inst = klass(**kwargs)
+            phases.append(phase_inst)
+            return phase_inst
```

Comment:
This change and the later `associate` assignment seem to be unused, maybe from some earlier work in progress?  Could you take them out please?